pkg/bootstrap/docker/up.go

```diff
@@ -45,6 +46,9 @@ const (
 	initialProjectDisplay = "My Project"
 	initialProjectDesc    = "Initial developer project"
 
+	defaultRedirectClient = "openshift-web-console"
+	defaultRedirectURI    = "https://localhost:9000"
```

Comment:
nit: this isn't the default redirect URI, it's an additional allowed one... more a "developmentRedirectURI"